pkg/capabilities/consensus/ocr3/store.go

```diff
@@ -43,11 +43,9 @@ func (s *store) getN(ctx context.Context, requestIDs []string) ([]*request, erro
 	o := []*request{}
 	for _, r := range requestIDs {
 		gr, ok := s.requests[r]
-		if !ok {
-			return nil, fmt.Errorf("request with id %s not found", r)
+		if ok {
```

Comment:
@bolekk This can't fail any more -- can we remove the `err` from the return? (And maybe add a comment that it's a "best-effort" get that isn't guaranteed to return everything requested.

Reply:
(I'll approve in the meantime, this is a nit, so if you want to defer to later, please do)